#goal definition
#action command to send (should be in the form action_param1_param2_...)
string action
string[] params
---
#result definition
#exit status of sent action, event to publish to /PNPConditionEvent
int32 exit_status
string[] condition_event
VariableValue[] set_variables
---
#feedback definition
#action running smoothly
bool run
